docs/source/explanations/optimization/why_optimization_is_hard.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>You write the algorithm failed, but the output says <code>terminated succesfully</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/estimagic/pull/335/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
That's the point. We know it failed because we know the result but the algorithm thinks it was successful because some convergence criterion is fulfilled. Will try to make it clearer.

Reply:
Ah, I see now. So actually whether the algorithm knows if it fails, depends on the seed. Fixed a seed now.